build.sbt

```diff
@@ -21,8 +21,16 @@ inThisBuild(
         "scm:git:git@github.com:scalameta/metadoc.git"
       )
     ),
+    publishTo := {
+      val nexus = "https://oss.sonatype.org/"
+      if (isSnapshot.value)
+        Some("snapshots" at nexus + "content/repositories/snapshots")
+      else
+        Some("releases"  at nexus + "service/local/staging/deploy/maven2")
+    },
     commands += Command.command("ci-release") { s =>
-      "very publishSigned" ::
+      s";^ publishSigned" ::
+        s";^^ ${Version.sbt013}; sbtPlugin/publishSigned" ::
```

Comment:
This could also be `+ publishSigned`, however, it won't map over both scala and sbt versions.